bin/strings.swift

```diff
@@ -120,7 +120,7 @@ let stringsByLocale = stringsByLocale1
     }
 }
 
-let supportedLocales = ["de", "en", "es", "fr", "ja"]
+let supportedLocales = ["Base", "de", "en", "es", "fr", "ja"]
```

Comment:
Note the two changes to this file:
- `en` gets mapped to `Base` in line 115, which is why I noticed that the script wasn't writing changes to the english `.strings` file. This is not a great solution, but honestly I don't think we should attempt any more improvements to this file until we can move it to SPM and write some tests for it
- second change only appends the locale comment if the locale is in the list of supported locales (removed `**it**:` etc from the `Strings.swift` file)

Reply:
Ah interesting - so this would have a regression introduced in #528?

Reply:
Yep exactly